src/app_service/service/accounts/dto/create_account_request.nim

```diff
@@ -21,6 +21,8 @@ type
 
     wakuV2Nameserver*: Option[string]
     wakuV2LightClient*: bool
+    wakuV2EnableStoreConfirmationForMessagesSent*: bool
+    wakuV2EnableMissingMessageVerification*: bool
```

Comment:
@richard-ramos I think you missed to add these fields to `toJson` function below 🙁